src/presenter/main-presenter.js

```diff
@@ -0,0 +1,57 @@
+import EventListView from '../view/event-list-view';
+import EventItemView from '../view/event-item-view';
+import SortView from '../view/sort-view';
+import FilterView from '../view/filter-view';
+import EditEventItemView from '../view/event-item-edit-view';
+import TripInfoView from '../view/trip-info-view';
+import { render, RenderPosition } from '../render';
+
+export default class MainPresenter {
+  eventListComponent = new EventListView;
+
+  constructor({container}) {
+    this.container = container;
+
+    this.sitePageBodyElement = document.querySelector('.page-body');
+    this.siteTripMainElement = this.sitePageBodyElement.querySelector('.trip-main');
+    this.siteControlsFiltersElemet = this.siteTripMainElement.querySelector('.trip-controls__filters');
+
+    this.siteTripEventsElement = this.sitePageBodyElement.querySelector('.trip-events');
```

Comment:
Не должно быть обращения к DOM в презентерах.

```suggestion
    this.sitePageBodyElement = document.querySelector('.page-body');
    this.siteTripMainElement = this.sitePageBodyElement.querySelector('.trip-main');
    this.siteControlsFiltersElemet = this.siteTripMainElement.querySelector('.trip-controls__filters');

    this.siteTripEventsElement = this.sitePageBodyElement.querySelector('.trip-events');
```